feature/gnoi/system/tests/supervisor_switchover_test/supervisor_switchover_test.go

```diff
@@ -180,10 +180,14 @@ func TestSupervisorSwitchover(t *testing.T) {
 
 	t.Log("Validate OC Switchover time/reason.")
 	activeRP := gnmi.OC().Component(rpActiveAfterSwitch)
-	if got, want := gnmi.Lookup(t, dut, activeRP.LastSwitchoverTime().State()).IsPresent(), true; got != want {
-		t.Errorf("activeRP.LastSwitchoverTime().Lookup(t).IsPresent(): got %v, want %v", got, want)
+
+	swTime, swTimePresent := gnmi.Watch(t, dut, activeRP.LastSwitchoverTime().State(), 1*time.Minute, func(val *ygnmi.Value[uint64]) bool { return val.IsPresent() }).Await(t)
+	if !swTimePresent {
+		t.Errorf("activeRP.LastSwitchoverTime().Watch(t).IsPresent(): got %v, want %v", false, true)
 	} else {
-		t.Logf("Found activeRP.LastSwitchoverTime(): %v", gnmi.Get(t, dut, activeRP.LastSwitchoverTime().State()))
+		st, _ := swTime.Val()
+		t.Logf("Found activeRP.LastSwitchoverTime(): %v", st)
+		// TODO: validate that last switchover time is correct
```

Comment:
hey @LimeHat,` gnmi.Get(t, dut, activeRP.LastSwitchoverTime().State())` in the logging statement already does the validation of the switchover time partially and removing that will decrease the test coverage. Can u please keep the same logging statement as before : `t.Logf("Found activeRP.LastSwitchoverTime(): %v", gnmi.Get(t, dut, activeRP.LastSwitchoverTime().State()))`

Reply:
I'm not following, could you elaborate. The log is still there,  you get the same output with my changes. What exactly is missing?

TODO statement only indicates the fact that currently we check the presence of this leaf but not the actual value (e.g. an implementation could report January 2nd 1970 each and every time and it is not automatically validated). I didn't make any changes in this area.

Reply:
you are right, i did not noticed the code above the logging statement.